editions/tw5.com/tiddlers/nodejs/Installing a plugin - client server.tid

```diff
@@ -0,0 +1,38 @@
+created: 20220611123344385
+modified: 20220612091241651
+tags: [[TiddlyWiki on Node.js]] PluginsCS
+title: Installing a plugin - client server
+
+Follow these instructions when using TiddlyWiki under client-server (Node.js) configuration:
```

Comment:
We need to be careful about nomenclature here. The client-server configuration is one of the configurations supported by the Node.js edition of TW5. The wording here gives the impression that Node.js is a synonym for client-server.